aspnetcore/aspnetcore-2.0.md

```diff
@@ -0,0 +1,285 @@
+
+For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/3385).
+
+## WebListener renamed to HttpSys
+
+The packages `Microsoft.AspNetCore.Server.WebListener` and `Microsoft.Net.Http.Server` have been merged into a new package `Microsoft.AspNetCore.Server.HttpSys`. The namespaces have been updated to match.
+
+For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/2648).
+
+## Enhanced HTTP header support
+
+When using MVC to transmit a `FileStreamResult` or a `FileContentResult`, you now have the option to set an `ETag` or a `LastModified` date on the content you wish to transmit.  You can set these values on the returned content with code similar to the following:
+
+```csharp
+var data = Encoding.UTF8.GetBytes("This is a sample text from a binary array");
+var entityTag = new EntityTagHeaderValue("\"MyCalculatedEtagValue\"");
+return File(data, "text/plain", "downloadName.txt", lastModified: DateTime.UtcNow.AddSeconds(-5), entityTag: entityTag);
+```
+ 
+The file returned to your visitors will be decorated with the appropriate HTTP headers for the `ETag` and `LastModified` values.
+
+If an application visitor requests content with a Range Request header, ASP.NET will recognize that and handle that header. If the requested content can be partially delivered, ASP.NET will appropriately skip and return just the requested set of bytes.  You do not need to write any special handlers into your methods to adapt or handle this feature; it is automatically handled for you.
+
+## Hosting startup and Application Insights
+
+Hosting environments can now inject extra package dependencies and execute code during application startup, without the application needing to explicitly take a dependency or call any methods. This feature can be used to enable certain environments, such as debugging in Visual Studio or hosting in Azure App Service, to "light-up" features unique to that environment without the application needing to know ahead of time.
+
+This feature is used to automatically enable Application Insights diagnostics  in ASP.NET Core 2.0 applications when debugging in Visual Studio and (after opting in) when running in Azure App Services. As a result, the project templates no longer add Application Insights packages and code by default.
+
+For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/3389).
+
+## Automatic use of anti-forgery tokens
```

Comment:
don't we need an issue for this? I'm documenting it for Razor pages. I didn't know MVC controllers get this too.